### PR TITLE
DP-2705: Updating the CONTRIBUTING.md with hugo

### PR DESCRIPTION
**Pull request by Mara3l**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -64,6 +64,67 @@ The documentation is deployed using manually triggered GitHub workflows.
...
+        ---
+
+        {{< python "sdk.CatalogPermissionService.super_permissions_method" >}}
+
+        ## Example
+        ```python
+        # This method does something very cool, trust me bro
+        sdk.super_permissions_method(user_id = "demo_user", give_all_permission = True)
+        ```
+        ```
```
Comment (hkad98) [inline CONTRIBUTING.md:114]: Typo

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by hkad98 — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR415

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/228/ : FAILURE in 1m 11s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/378/ : SUCCESS in 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/382/ : SUCCESS in 12s

**Pull request merged**